code-studio/package.json

```diff
@@ -1,104 +0,0 @@
-{
-  "name": "@cdo/code-studio",
-  "version": "0.0.1",
-  "private": "true",
-  "description": "Pipeline for static assets used by our Code Studio rails app (i.e. dashboard)",
-  "author": "Code.org",
-  "license": "Apache-2.0",
-  "config": {
-    "engine-strict": true
-  },
-  "engines": {
-    "node": ">=0.12.0",
-    "npm": "^2.9.1"
-  },
-  "scripts": {
-    "postinstall": "linklocal",
-    "lint": "npm run lint-js && npm run lint-css",
-    "lint-css": "sass-lint src/css/*css",
-    "lint-js": "eslint src/js --ext .js,.jsx",
-    "test": "mochify test/**/*.js --extension=.jsx --transform babelify",
-    "debug": "mochify --debug test/**/*.js --transform babelify",
-    "coverage": "mochify test/**/*.js --transform babelify --plugin [ mochify-istanbul --report json --report lcov --dir ./coverage ] && istanbul check-coverage ./coverage/coverage-final.json",
-    "clean": "rm -rf build",
-    "clean-assets": "rm -rf build/assets",
-    "clean-css": "rm -rf build/css",
-    "clean-js": "rm -rf build/js",
-    "copy-assets": "scripts/copy-assets.js",
-    "build": "npm run copy-assets && npm run build-css && npm run build-js",
-    "build:dist": "npm run clean && npm run copy-assets && npm run build-css:dist && npm run build-js:dist",
-    "build-css": "scripts/build-css.js",
-    "build-css:dist": "npm run lint-css && npm run clean-css && scripts/build-css.js --dist",
-    "build-js": "scripts/build-js.js",
-    "build-js:dist": "npm run lint-js && npm test && npm run clean-js && scripts/build-js.js --dist && scripts/uglify-js.js",
-    "start": "scripts/build-js.js --watch",
-    "convert-scss-vars": "scripts/convert-scss-variables.js",
-    "prebuild-js": "npm run convert-scss-vars",
-    "prebuild-js:dist": "npm run convert-scss-vars",
-    "prestart": "npm run convert-scss-vars"
-  },
-  "//": "ejsify is a dependency of apps/src; we declare it here rather than in apps/src/package.json so we can continue to avoid needing to npm install that module.",
-  "devDependencies": {
```

Comment:
It's hard to tell at a glance whether all of these dependencies got moved to apps/package.json.  Do we need to do a careful audit of this?